src/edit.js

```diff
@@ -25,26 +31,70 @@ import Block from './block';
  * @see https://developer.wordpress.org/block-editor/developers/block-api/block-edit-save/#edit
  * @return {WPElement} Element to render.
  */
-export default function Edit( { attributes, setAttributes } ) {
+function ProgressEdit( {
+	attributes,
+	setAttributes,
+	messageColor,
+	setMessageColor,
+	progressColor,
+	setProgressColor,
+} ) {
 	const { freeShippingFrom } = attributes;
 
 	return (
 		<div { ...useBlockProps() }>
 			<InspectorControls>
 				<PanelBody title="Settings" initialOpen={ true }>
 					<TextControl
-						label="Free shipping from"
-						help="Provide the value in your store currency."
+						label={ __(
+							'Free shipping from',
+							'free-shipping-progress-bar'
+						) }
+						help={ __(
+							'Provide the value in your store currency.',
+							'free-shipping-progress-bar'
+						) }
 						value={ freeShippingFrom }
 						onChange={ ( value ) =>
 							setAttributes( {
 								freeShippingFrom: value,
 							} )
 						}
 					/>
```

Comment:
Outside scope of this PR, but I just saw this. This value is not persisting, but I won't tell you yet why 😉

Reply:
Oh, it seems I have defined the wrong attribute type. I changed the type from `number` to `string`. Now the variable gets stored as expected.

Reply:
The value should be however a number to do arithmetics on it.